src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery randomBall = new Lottery();
```

Comment:
Lottery is Lottery and Ball is Ball
Do not use 'ball' as a name for Lottery - it is very confusing